profiling/Cargo.toml

```diff
@@ -25,9 +27,10 @@ libc = "0.2"
 # TRACE set to max to support runtime configuration.
 log = { version = "0.4", features = ["max_level_trace", "release_max_level_trace"]}
 once_cell = { version = "1.12" }
-uuid = { version = "1.0", features = ["v4"] }
+ouroboros = { version = "0.15.6" }
```

Comment:
Need version 0.17+ to avoid a soundness bug:
https://github.com/joshua-maros/ouroboros/issues/88